packages/design-system-react-native/src/components/AvatarFavicon/AvatarFavicon.constants.ts

```diff
@@ -10,3 +10,14 @@ export const DEFAULT_AVATARFAVICON_PROPS: Required<
   width: '100%',
   height: '100%',
 };
+
+// Sample Favicon URIs
+export const SAMPLE_AVATARFAVICON_URIS = [
+  'https://metamask.github.io/test-dapp/metamask-fox.svg',
+  'https://www.coinbase.com/favicon.ico',
+  'https://www.myetherwallet.com/favicon.ico',
+  'https://www.blockchain.com/static/favicon.ico',
+  'https://trezor.io/favicon.ico',
+  'https://electrum.org/favicon.ico',
+  'https://cryptologos.cc/logos/ethereum-eth-logo.svg',
+];
```

Comment:
Added these so they can be reused for both `AvatarFavicon`, `AvatarGroup`, and any other components built from `AvatarFavicon`